### ISSUE TITLE
[React Native Gallery -> Flyout]: The text inside the button is not visible in Flyout card in high contrast themes.

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#308]
**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to ‘Flyout’ button and invoke it.
Observe 'Open Popup' button in high contrast Aquatic Theme and Desert Theme.
Invoke 'Open Popup' button and Observe 'Close Flyout' button in high contrast Aquatic Theme and Desert Theme. 

**Actual Result:**
The text inside the button is not visible in Flyout card in high contrast themes.

Note: Same issue is observed in Popup, Pressable, Touchable Highlight and Touchable Opacity cards. 

**Expected Result:**
All the content should be clearly visible in Flyout card in high contrast themes. 

**User impact:**
Users with low vision will not be able to consume the content present in the Flyout card if it is not visible in High Contrast Themes.

Internal bug link: https://dev.azure.com/microsoft/OS/_workitems/edit/43367578

**Comment by YajurG** [comment]
Main issue here is that the background color for the TouchableHighlight is too light - tried changing the background color to silver and it was visible in both Desert and Aquatic high contrast modes.
